IntegrationTests/src/bkr/inttest/client/test_system_modify.py

```diff
@@ -4,7 +4,8 @@
 # the Free Software Foundation; either version 2 of the License, or
 # (at your option) any later version.
 
-from bkr.server.model import session, SystemStatus, SystemPermission, Hypervisor
+from bkr.server.model import session, SystemStatus, SystemPermission, Hypervisor, \
```

Comment:
Do not use `\`. Instead of that, you can use `(` and `)`

Reply:
Done